info.femm.unofficial-manifest.yml

```diff
@@ -0,0 +1,234 @@
+app-id: info.femm.unofficial
```

Comment:
Missed this the first time, but the manifest file name doesn't match the app-id.
https://github.com/flathub/flathub/wiki/App-Requirements#repository-layout